interface/modules/custom_modules/oe-module-payroll/public/provider_rate/postupdate.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ *  package   OpenEMR
+ *  link      http://www.open-emr.org
+ *  author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  copyright Copyright (c )2021. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ */
+
+namespace Juggernaut\Modules\Payroll;
```

Comment:
shouldn't above be a `use` rather than a `namespace` to bring in the ProviderRates() class?

Reply:
I chose to do namespace instead of use. It works either way. I was trying it out.

Reply:
rec using `use` :)